Alignment/MillePedeAlignmentAlgorithm/python/mpsvalidate/iniparser.py

```diff
@@ -0,0 +1,318 @@
+
+# Read the ini data which is passed to the function and return the
+# data as a configData object. If a parameter is given the function
+# parseParameter will override the config values.
+##
+
+import ConfigParser
+import logging
+import os
+
+
+class ConfigData:
+    """ stores the config data of the ini files or the console parameters
+    """
+
+    def __init__(self):
+        # get path to modules, defaut ini and templates
+        self.mpspath = os.path.join(os.path.dirname(__file__))
+        
+        # General
+        # jobmX dir
+        self.jobNumber = -1
+        # MillePedeUser_X time
+        self.jobTime = -1
+        # ./jobData/jobmX path
+        self.jobDataPath = ""
+        # base outputpath
+        self.outputPath = ""
+        # latex file name
+        self.latexfile = ""
+        # identification in every plot (e.g. mp1885)
+        self.message = ""
+        # limits for warning dict with keys xyz, rot, dist
+        # arguments must be given in this order
+        self.limit = {}
+        # statboxsize
+        self.statboxsize = -1
+
+        # what should be created
+        self.showmonitor = -1
+        self.showadditional = -1
+        self.showdump = -1
+        self.showtime = -1
+        self.showhighlevel = -1
+        self.showmodule = -1
+        self.showsubmodule = -1
+        self.showtex = -1
+        self.showbeamer = -1
+        self.showhtml = -1
+
+        # MODULEPLOTS
+        # number of bins after shrinking
+        self.numberofbins = -1
+        # definition of sharp peak; max_outlier / StdDev > X
+        self.defpeak = -1
+        # new histogram width in units of StdDev
+        self.widthstddev = -1
+        # every parameter (e.g. xyz) with same range
+        self.samerange = -1
+        # rangemode "stddev" = multiple of StdDev, "all" = show all, "given" =
+        # use given ranges
+        self.rangemode = -1
+        # ranges
+        self.rangexyzM = []
+        self.rangerotM = []
+        self.rangedistM = []
+
+        # HIGHLEVELPLOTS
+        # given ranges
+        self.rangexyzHL = []
+        self.rangerotHL = []
+        # every parameter (e.g. xyz) with same range
+        self.samerangeHL = -1
+        # rangemode "all" = show all, "given" = use given ranges
+        self.rangemodeHL = -1
+
+        # Time dependent
+        self.firsttree = -1
+
+        # list with the plots for the output
+        self.outputList = []
+
+    def parseConfig(self, path):
+        logger = logging.getLogger("mpsvalidate")
+        
+        # create ConfigParser object
+        parser = ConfigParser.ConfigParser()
+
+        # read ini file
+        if (parser.read(path) == []):
+            logger.error("Could not open ini-file: {0}".format(path))
+
+        # buffer object
+        configBuffer = ConfigData()
+
+        # collect data and process it
+        try:
+            configBuffer.jobNumber = int(parser.get("GENERAL", "job"))
+        except:
```

Comment:
@prigoll here and below you are catching all exceptions w/o specifying what exceptions you expect here, i.e. you will also silently catch exceptions that you do not expect;
please be explicit here